src/sqs/index.ts

```diff
@@ -100,13 +150,16 @@ export class SqsQueue<MessagePayload> {
 	 */
 	listenerContext: ListenerContext;
 
-	public constructor(settings: QueueSettings, messageHandler: MessageHandler<MessagePayload>) {
+	public constructor(settings: QueueSettings, messageHandler: MessageHandler<MessagePayload>, errorHandler: ErrorHandler<MessagePayload>) {
```

Comment:
handing an interface with a handler function in it seems excessive.  Couldn't it just be the function alone?